### PR TITLE
Nonleaders can suggest variants using dropdown

### PR DESCRIPTION
**Pull request by newduke**
No issue. I just found it annoying that suggesting variants as a nonleader I would have to remember variant names exactly but there is already a perfectly good UI for that.

### CODE REVIEW

File: client/src/chat.ts [hunk 1]
```diff
@@ -220,8 +219,12 @@ function send(room: string, element: JQuery<HTMLElement>) {
   // Clear the chat box
   element.val("");
 
+  sendText(room, msg);
+}
+
+export function sendText(room: string, _msg: string): void {
```
Comment (Zamiell) [inline client/src/chat.ts:225]: get rid of the underscore

Comment (newduke) [reply client/src/chat.ts:225]: Lint complains about reassigning parameters. Prefer to rename to msgConst or something?

Comment (Zamiell) [reply client/src/chat.ts:225]: if you have to reassign a parameter, then make a new variable that has a name describing how it was modified

Comment (newduke) [reply client/src/chat.ts]: I dislike `msgWithEmojis`.

Comment (Zamiell) [reply client/src/chat.ts]: maybe formattedMsg?

Comment (newduke) [reply client/src/chat.ts]: Sure, but the only format changed is that last emoji... it really is `msg` so I think `msgRaw` is smart naming

Comment (newduke) [reply client/src/chat.ts]: I guess it's less pretty as an export

Comment (Zamiell) [reply client/src/chat.ts]: i dont feel strongly either way

File: client/src/chat.ts [hunk 2]
```diff
@@ -208,7 +208,6 @@ function send(room: string, element: JQuery<HTMLElement>) {
     throw new Error("The value of the element is not a string.");
   }
   msg = msg.trim();
-
```
Comment (Zamiell) [inline client/src/chat.ts:211]: why remove this

File: client/src/lobby/nav.ts [hunk 3]
```diff
@@ -148,12 +149,16 @@ export function init(): void {
         if (VARIANTS.get(variantName) === undefined) {
           return;
         }
-        globals.conn!.send("tableSetVariant", {
-          tableID: globals.tableID,
-          options: {
-            variantName,
-          },
-        });
+        if (globals.game!.owner === globals.userID) {
```
Comment (Zamiell) [inline client/src/lobby/nav.ts:152]: probably better written as `game?` right?

Comment (newduke) [reply client/src/lobby/nav.ts:152]: Fine by me.

**Comment by Zamiell** [comment]
thx ill test this out later

**Pull request merged**